internal/util/crypto.go

```diff
@@ -196,40 +196,44 @@ func VolumeMapper(volumeID string) (mapperFile, mapperFilePath string) {
 
 // EncryptVolume encrypts provided device with LUKS.
 func EncryptVolume(ctx context.Context, devicePath, passphrase string) error {
-	log.DebugLog(ctx, "Encrypting device %s with LUKS", devicePath)
-	if _, _, err := LuksFormat(devicePath, passphrase); err != nil {
-		return fmt.Errorf("failed to encrypt device %s with LUKS: %w", devicePath, err)
+	log.DebugLog(ctx, "Encrypting device %q	 with LUKS", devicePath)
+	_, stdErr, err := LuksFormat(devicePath, passphrase)
+	if err != nil || stdErr != "" {
+		log.ErrorLog(ctx, "failed to encrypt device %q with LUKS (%v) %s", devicePath, err, stdErr)
```

Comment:
lol, almost the same nit as before: I'd prefer to have `(%v): %s` with the `:` after `(%v)`... You had it earlier, but dropped it  as well when I asked to drop the `:` _before_ `(%v)`.

Reply:
:D pushed new changes. need approval again